### DISCUSSION TITLE
Game Server Console Refresh Rate

### DISCUSSION THREAD

**Discussion by Lazylllama** [gameserverapp/Platform#529] [Suggestions]
The server console is **EXTREMELY** slow at refreshing. The command is executed instantly but it takes over 3-5 seconds to get a response. Due to this i cant use it and i rather use a RCON client.

**Comment by mvaessen** [comment]
Hi! There is indeed a small delay with the responses showing up. This is by design due to the scale GSA is operating at.
Whether a couple seconds delay is "extremely slow", is probably subjective.  I personally would consider a couple minutes to be "extremely slow".

Either way; if a couple seconds delay is too much for whatever you are trying to do with the console, I do recommend using a RCON client instead as you mentioned.
